src/huggingface_hub/hf_api.py

```diff
@@ -3612,7 +3818,8 @@ def create_commit(
             revision=unquoted_revision,  # first-class methods take unquoted revision
             create_pr=create_pr,
             num_threads=num_threads,
-            free_memory=False,  # do not remove `CommitOperationAdd.path_or_fileobj` on LFS files for "normal" users
+            # do not remove `CommitOperationAdd.path_or_fileobj` on LFS files for "normal" users
+            free_memory=False,
```

Comment:
```suggestion
            free_memory=False,  # do not remove `CommitOperationAdd.path_or_fileobj` on LFS files for "normal" users
```
(something might be off with your `make style` command :thinking: )

Reply:
@Wauplin haven't changed anything in the makefile, so it automatically formats these comments over the code, I'll revert those lines